src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -406,5 +406,40 @@ internal static TypeSymbol GetTypeOrReturnType(this Symbol symbol)
                     throw ExceptionUtilities.UnexpectedValue(symbol.Kind);
             }
         }
+
+        internal static bool IsWellKnownTypeIsConst(this ISymbol symbol)
```

Comment:
It can use `ITypeSymbol`. Moved to `TypeSymbolExtensions`

---
In reply to: [119750837](https://github.com/dotnet/roslyn/pull/19658#discussion_r119750837) [](ancestors = 119750837)